.github/workflows/E2Etests.yml

```diff
@@ -0,0 +1,70 @@
+name: End-to-end tests
+on: [push]
+
+jobs:
+  cypress-e2e-headless:
+    name: Cypress e2e
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: ["14"]
+        browser: ["firefox", "chrome"]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup and Run backend
+        run: |
+          git clone --branch Added_test_docker-compose https://github.com/CSCfi/beacon-network
```

Comment:
i think it would be good if we add this branches to at least `dev` branch in `beacon-network` repo.

Reply:
Yes i'm currently updating that branch